src/cata_tiles.cpp

```diff
@@ -680,34 +680,41 @@ void cata_tiles::load_ascii_set( JsonObject &entry, int offset, int size )
 
 void cata_tiles::load_tilejson_from_file( JsonObject &config, int offset, int size )
 {
-    if( !config.has_member( "tiles" ) ) {
+    if ( !config.has_member( "tiles" ) ) {
```

Comment:
Seriously, not cleaning up is one thing, but you or your editor are consistently breaking styling.